dimod/reference/samplers/exact_solver.py

```diff
@@ -85,6 +86,10 @@ def sample(self, bqm):
             :obj:`~dimod.SampleSet`
 
         """
+        for kw in kwargs:
+            msg = "Ignoring unknown kwarg: {!r}".format(kw)
+            warnings.warn(msg, RuntimeWarning)
```

Comment:
We probably also want to increase the stacklevel so that the warning shows where the `.sample()` call is.